frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -175,6 +181,13 @@ const getColumns = (t: TFunction): TableColumn<NodeRowItem>[] => [
     transforms: [sortable],
     props: { className: nodeColumnInfo.cpu.classes },
   },
+  {
+    title: t('console-app~Architecture'),
+    id: nodeColumnInfo.architecture.id,
+    sort: sortWithCSRResource(nodeArch, ''),
+    transforms: [sortable],
+    props: { className: nodeColumnInfo.architecture.classes },
```

Comment:
```suggestion
    props: { className: nodeColumnInfo.architecture.classes },
     additional: true,
```

Reply:
![image](https://github.com/user-attachments/assets/5e5fa702-73a8-486a-ba4d-55b37aae7351)

Reply:
Yes , it works . thanks @prb112